examples/FMO dynamics with Redfield theory.ipynb

```diff
@@ -464,31 +120,29 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
-   "metadata": {
-    "collapsed": false
-   },
+   "execution_count": 20,
+   "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "[ 0.02394464  0.01493232  0.68821011  0.22268562  0.02374941  0.00193277\n",
-      "  0.02454513]\n",
-      "[ 0.02408245  0.0149764   0.68788931  0.22268262  0.02380367  0.00195459\n",
-      "  0.02461096]\n"
+      "[0.02394464 0.01493232 0.68821011 0.22268562 0.02374941 0.00193277\n",
+      " 0.02454513]\n",
+      "[0.02408245 0.0149764  0.68788931 0.22268262 0.02380367 0.00195459\n",
+      " 0.02461096]\n"
      ]
     }
    ],
    "source": [
-    "print np.diag(ket_vec_to_matrix(rho[-1])).real\n",
+    "print np.diag(qs.ket_vec_to_matrix(rho[-1])).real\n",
     "\n",
     "import scipy\n",
     "def thermal_state(hamiltonian, temperature):\n",
     "    rho = scipy.linalg.expm(-hamiltonian.in_rotating_frame().H_1exc / temperature)\n",
     "    return rho / np.trace(rho)\n",
     "\n",
-    "print np.diag(thermal_state(fmo_hamiltonian, CM_K * 77)).real"
+    "print np.diag(thermal_state(fmo_hamiltonian, qs.CM_K * 77)).real"
```

Comment:
can we call `fmo_hamiltonian.thermal_state(qs.CM_K * 77)` instead? I don't think we need to use the separate helper function.

Reply:
changed in 1e152be4893b5e31d9f21c1bf9860c90643c2717
`fmo_hamiltonian.thermal_state(subspace)` temperature is defined by the bath